templates/server-statefulset.yaml

```diff
@@ -83,11 +83,11 @@ spec:
           {{ template "vault.mounts" . }}
           ports:
             - containerPort: 8200
-              name: http
+              name: {{ include "vault.scheme" . }}
             - containerPort: 8201
-              name: internal
+              name: {{ include "vault.scheme" . }}-internal
             - containerPort: 8202
-              name: replication
+              name: {{ include "vault.scheme" . }}-rep
```

Comment:
fyi, this is "rep", not "replication", because the port name must be under <=15 characters